presentation/src/main/java/com/mashup/alcoholfree/presentation/ui/measuring/MeasuringScreen.kt

```diff
@@ -0,0 +1,211 @@
+private val measuringShape = RoundedCornerShape(16.dp)
+
+@Composable
+fun MeasuringScreen(
+    state: MeasuringState,
+    onMeasureFinishClick: () -> Unit = {},
+    onBackButtonClick: () -> Unit = {},
+) {
+    Box(
+        modifier = Modifier.fillMaxSize(),
+    ) {
+        SulSulBackButton(
+            modifier = Modifier
+                .padding(top = 8.dp, start = 16.dp)
+                .align(Alignment.TopStart),
+            onClick = onBackButtonClick,
+        )
+
+        Column(
+            modifier = Modifier.fillMaxSize(),
+            horizontalAlignment = Alignment.CenterHorizontally,
+        ) {
+            MeasuringHeader(
+                modifier = Modifier.padding(top = 38.dp),
+                totalCount = state.totalCount,
+                status = state.records,
+            )
+
+            if (state.totalCount > 0) {
+                SulSulLargeBadge(
+                    modifier = Modifier.padding(top = 16.dp),
+                    type = SulSulBadgeType.PURPLE,
+                    text = state.level,
+                )
+            }
+
+            MeasuringSelectedAlcoholBubble(
+                modifier = Modifier.padding(vertical = 16.dp),
+                bubbleResId = state.currentAlcoholType.iconResId,
+            )
+
+            MeasuringBubblesContainer(
+                modifier = Modifier.weight(1f),
+            )
+
+            MeasuringAlcoholSelection(
+                modifier = Modifier.padding(horizontal = 16.dp, vertical = 12.dp),
+            )
+
+            MeasuringFinishButton(
+                modifier = Modifier.padding(bottom = 40.dp),
+                onMeasureClick = onMeasureFinishClick,
+            )
+        }
+    }
+}
+
+
+@Composable
+private fun MeasuringHeader(
+    modifier: Modifier = Modifier,
+    totalCount: Int,
+    status: String,
+) {
+    Column(
+        modifier = modifier,
+        horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+        Text(
+            text = stringResource(id = R.string.alcohol_cup_count, totalCount),
+            style = H1,
+            color = White,
+        )
+        Text(
+            modifier = Modifier.padding(top = 4.dp),
+            text = status,
+            style = SubTitle3,
+            color = White,
+        )
+    }
+}
+
+@Composable
+private fun MeasuringAlcoholSelection(
+    modifier: Modifier = Modifier,
+) {
+    Box(
+        modifier = modifier
+            .fillMaxWidth()
+            .clip(measuringShape)
+            .background(color = Grey100)
+            .border(width = 1.dp, color = Grey300, shape = measuringShape),
+        contentAlignment = Alignment.Center,
+    ) {
+        /* TODO: 주종 선택 컴포넌트 */
+        Text(
+            modifier = Modifier.padding(vertical = 8.dp),
+            text = stringResource(id = R.string.alcohol_type_selection),
+            style = SubTitle3,
+            color = Grey800,
+        )
+    }
+}
+
+@Composable
+private fun MeasuringSelectedAlcoholBubble(
```

Comment:
```suggestion
private fun MeasuringAlcoholBubble(
```

Nit: 용도를 궂이 함수 이름에 적어주지 않아도 괜찮지않을까!? 조금 더 범용적으로 사용해도 괜찮은거같아서..!

Reply:
오 좋은 것 같아용! 근데 저 버블 생각해보니까 터치하면 떨어지는거라 컨테이너 영역에 포함되어야 할 것 같다는 생각이 갑자기 드네여..?

Reply:
그런거같다,, 아에 제거해둬도 될거같기도..?